Sources/Core/BasePresenter.swift

```diff
@@ -25,26 +25,40 @@
 //
 import UIKit
 
-open class BasePresenter: OnboardingPresenter {
+public struct OnboardingSlide {
+
+    let titleText: String
+    let bodyText: String
+    let image: UIImage?
 
-    public typealias OnboardingData = (titleText: String, bodyText: String, image: UIImage?)
+    public init(titleText: String, bodyText: String, image: UIImage?) {
```

Comment:
The default has internal level of protection, I had to add that in order to use it from the outside of the framework